### DISCUSSION TITLE
simple-bar now supports AeroSpace WM

### DISCUSSION THREAD

**Discussion by Jean-Tinland** [nikitabobko/AeroSpace#910] [general]
Hi!

I don't know if this is okay, but I just wanted to let people know that the project I work on, [simple-bar](https://github.com/Jean-Tinland/simple-bar), is now supporting AeroSpace.

That said, thank you, @nikitabobko, for this great tool; it is very convenient to use!

Feel free to delete this discussion if it is out of topic or considered self-promotion!

**Comment by nikitabobko** [comment]
Yes, it's totally ok. Thank you for your work!

I even have a separate page for 3rd party integrations https://nikitabobko.github.io/AeroSpace/goodies
You are welcome to create the PR to `./docs/goodies.adoc` (or I can create the PR myself) to add `simple-bar` to the page. But I think it worth to add a small disclaimer that the focused workspace isn't instantly updated after the workspace switch https://github.com/Jean-Tinland/simple-bar/issues/423 as I think it might a dealbreaker for a lot of people.

**Comment by Jean-Tinland** [comment]
Thanks! That's great I'll create a PR when I'll be sure my implementation is fully working and is stable.

Yeah this delay thing is a pain point that often came up since the beginning of simple-bar, I'll make sure that it is clear that there are some limitation due to the way simple-bar works.
